UserAPI/Tags.md

```diff
@@ -25,4 +25,12 @@ Name | Permission
 
 ## Trust Levels
 
-TODO
+
+Field | Type
+------|-----
+<none\>|Visitor
+system_trust_basic|New User
+system_trust_known|User
+system_trust_trusted|Known User
+system_trust_veteran|Trusted User
+system_trust_legend|Veteran User (removed)
```

Comment:
It's only removed from the client, still exists from the API's point of view.

Reply:
yes
should I remove the `(removed)` text ?

Reply:
Replace it with `Unused`

Reply:
done ^^